repose-aggregator/src/docs/asciidoc/filters/cors.adoc

```diff
@@ -26,18 +33,45 @@ All of these headers will be supplied by the client making the request.
 | None required
 a| * `Origin`
    * `Access-Control-Request-Method`
-a| * `Origin`
+a| * `Origin`*
 |===
 
-=== Created
-The CORS filter wil not add any headers to the response for non-CORS requests.
-The filter will add the following headers to the response for these scenarios when the request is allowed to proceed:
+*{sp}If the `Origin` header matches the original `Host` header in the request, it will be considered a non-CORS request.
```

Comment:
Nope. The HTML itself is all wonky (as you would probably expect). So the Asciidoctor HTML engine probably needs to be changed. Well, or we get straight up cray-cray with CSS. But no thanks.